lib/main.dart

```diff
@@ -63,17 +64,18 @@ Future<void> main() async {
   await Localization.ensureInitialized();
   await SystemChrome.setPreferredOrientations([DeviceOrientation.portraitUp]);
 
-//Todo if catcher is not picking up error and exception then we should switch to sentryFlutter
-// SentryFlutter.init((options) {
-//   options.debug = true;
-//   options.anrEnabled = true;
-//   options.autoInitializeNativeSdk = true;
-//   options.attachScreenshot = true;
-//   options.dsn = Platform.isAndroid
-//       ? 'https://4753d78f885f4b79a497435907ce4210@o75725.ingest.sentry.io/5850353'
-//       : 'https://c14296fdf5a6be272e1ecbdb7cb23f76@o75725.ingest.sentry.io/4506081382694912';
-// }, appRunner: () => setupCatcherAndRun(LanternApp()));
-  setupCatcherAndRun(const LanternApp());
+  SentryFlutter.init((options) {
```

Comment:
Seems like using their sentry setup reports native [crashes](https://getlantern.sentry.io/issues/5686021697/?project=4507734480912384&query=&referrer=issue-stream&statsPeriod=14d&stream_index=1), also it provides lot more info it on [exception](https://getlantern.sentry.io/issues/5686025789/?project=4507734480912384&query=&referrer=issue-stream&statsPeriod=14d&stream_index=0)

Reply:
Nice! Thanks for making this change.